rabbitsmpp/consumer.go

```diff
@@ -2,29 +2,53 @@ package rabbitsmpp
 
 import (
 	"encoding/json"
-	"errors"
 	"log"
+	"github.com/vektra/errors"
```

Comment:
Since this is an open-source project, we should keep external dependencies to the minimum :)